### PR TITLE
Fix/readme update

### PR DESCRIPTION
**Pull request by bengarvey**
This changes the readme to reflect the changes in the other PRs.

##### Functional tests
- Run the new table commands and make sure they execute

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ CREATE TABLE `customer_entity` (
 CREATE TABLE `sales_flat_order` (
   `entity_id` int UNSIGNED NOT NULL auto_increment COMMENT 'Entity Id' ,
   `grand_total` decimal(12,4) NULL COMMENT 'Grand Total' ,
+  `base_grand_total` decimal(12,4) NULL COMMENT 'Base Grand Total' ,
```
Comment (ynicolaou) [inline README.md]: Add `base_discount_amount`.

Comment (bengarvey) [inline README.md]: There's another ticket to add the discount amount and it's not quite as simple. I can add a blank one though.

Comment (ynicolaou) [inline README.md]: Yea, i think a blank one would be fine for now.

**Comment by bengarvey** [comment]
I'm going to do that in another PR

**Pull request merged**